_docs/geyser/anticheat-compatibility.md

```diff
@@ -14,14 +14,15 @@ Full Compatibility (Checks Bedrock Players Accurately)
 - [Themis](https://www.spigotmc.org/resources/90766/){:target="_blank"} - Last Updated on 15th Jun 2022
 - [Negativity V2](https://www.spigotmc.org/resources/86874/){:target="_blank"} (Paid) - Last Updated on 21st March 2022
 - [SoaromaSAC](https://www.spigotmc.org/resources/87702/){:target="_blank"} - Last Updated on 27th March 2022
+- [Spartan](https://www.spigotmc.org/resources/25638/){:target="_blank"} (Paid) - Last Updated on 3rd January 2021
+
 
 Partially Compatible (Does not Check/Ignores Bedrock Players)
 
 - [AAC](https://www.spigotmc.org/resources/6442/){:target="_blank"} (Paid) - Last Updated on 19th August 2021
 - [Flappy Anticheat](https://www.spigotmc.org/resources/92180/){:target="_blank"} - Last Updated on 14th September 2021
-- [GrimAC](https://github.com/MWHunter/Grim){:target="_blank"} - Last Updated on 24th March 2022
+- [GrimAC](https://www.spigotmc.org/resources/99923/){:target="_blank"} (Unsupported/Abandoned) - Last Updated on 3rd January 2022
```

Comment:
If it's abandoned, do we really still want it here?

Reply:
+ from their GitHub page, it doesn't seem like it's abandoned.. it was updated to 1.19.3 recently

Reply:
Grim's current state of being maintained isn't exactly clear, over the last month the author claimed to abandon it, then subsequently locked the Discord and Github issues, however the Github has recently reopened and some updates were made, so it's unclear.

See: https://github.com/GrimAnticheat/Grim/commit/bdd5cbe7b26d66190fe1debea04acaad12c09419

Regardless, for normal users, there is no way to receive support for the plugin right now, so I think it's a good idea to warn users of that before they make their decision, however I don't think it should be removed because it's still receiving somewhat regular updates and is actively recommended and used

Reply:
I'd just say "Unsupported" then (and update the year too)

Reply:
done👍